owlbot.py

```diff
@@ -26,29 +27,46 @@
 #  Get api-common-protos
 # ----------------------------------------------------------------------------
 
-# Use api-common-protos as a proper synthtool git source
+# Clean up api-common-protos
+shutil.rmtree('api-common-protos', ignore_errors=True)
+
+# Clone api-common-protos
 api_common_protos_url = git.make_repo_clone_url(API_COMMON_PROTOS_REPO)
-api_common_protos = git.clone(api_common_protos_url)
+subprocess.run(["git", "clone", api_common_protos_url])
+
+# This is required in order for s.copy() to work
+s._tracked_paths.add("api-common-protos/google")
```

Comment:
It looks like this reaches into a attribute that's intended to be private. Is there another way to do this?

Reply:
As per discussion, this is a work around for compatibility with synthtool in order to use `s.copy()`